vicodyn/include/cocaine/vicodyn/peer.hpp

```diff
@@ -82,17 +83,26 @@ class peer_t : public std::enable_shared_from_this<peer_t> {
 // thread safe wrapper on map of peers indexed by uuid
 class peers_t {
 public:
-    class app_service_t {
-        using clock_t = std::chrono::steady_clock;
+    using clock_t = std::chrono::steady_clock;
 
+    class app_service_t {
         clock_t::time_point ban_until_;
+        std::unique_ptr<metrics::usts::ewma<clock_t>> timings_ewma_;
```

Comment:
Why is it stored as pointer? Are you going to `reset` it?

Reply:
Because metrics::usts::ewma has not copy and move constructor (because contains std::atomic<>). I can use pointer instead of app_service_t value or add constructors to metrics::usts::ewma in another repository.